src/Components/People.js

```diff
@@ -0,0 +1,82 @@
+import { Component } from "react";
+import axios from "axios";
+
+import "./People.css"
+
+class People extends Component {
+  state = {
+    input: "",
+    personSearchArray: [],
+    hasSearched: false,
+  };
+
+  handleChange = (e) => {
+    this.setState({ input: e.target.value });
+    this.setState({ hasSearched: true });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    this.personSearch();
+  };
+
+  personSearch = async () => {
+    try {
+      const res = await axios.get("https://ghibliapi.herokuapp.com/people");
+      this.setState({ personSearchArray: res.data });
+    } catch (error) {
+      this.setState({ personSearchArray: [] });
+    }
+  };
+
+
+  render() {
+    const { input, personSearchArray } = this.state;
+
+    return (
+      <div>
+        <h1>Search for a Person</h1>
+        <form onSubmit={this.handleSubmit}>
+          <input className="inputField"
+            type="text"
+            value={input}
+            onChange={this.handleChange}
+            placeholder="Find Your Person"
+          ></input>
+          <button type="submit" className="submitButton">Submit</button>
+        </form>
+        {}
+        <section className="personInfo">
+          {personSearchArray.map((person) => {
```

Comment:
Instead of mapping we could use, filter or better find, before the return and then add a little logic to see if we actually found someone or if we should be putting "Not Found"